changes/2589.changed

```diff
@@ -0,0 +1,10 @@
+Updated all screenshots on the README.md to gifs.
+----
+1. /nautobot/docs/media/nautobot_chatops.gif
+2. /nautobot/docs/media/nautobot_graphql.gif
+3. /nautobot/docs/media/nautobot_golden_config.gif
+4. /nautobot/docs/media/nautobot_mainpage.gif
+5. /nautobot/docs/media/nautobot_prefix_hierarchy.gif
+6. /nautobot/README.md
+7. nautobot/docs/media/nautobot_config_context.gif
+----
```

Comment:
This will create separate changelog lines. Thanks for the verbosity but not necessary.
```suggestion
```